compiling for grpc gem fails in rb_call.c

Using Ruby 2.2.1 installed via rbenv.
Following instructions from http://www.grpc.io/docs/installation/ruby.html
brew-based script installer fails:
```bash
~/.rbenv/versions/2.2.1/bin/ruby -r ./siteconf20150601-56693-1w5nx19.rb extconf.rb --with-grpc-dir=/usr/local
checking for grpc_channel_destroy() in -lgrpc... yes
checking for grpc_channel_destroy() in -lgrpc... yes
checking for gpr_now() in -lgpr... yes
creating Makefile

make "DESTDIR=" clean

make "DESTDIR="
compiling rb_byte_buffer.c
compiling rb_call.c
rb_call.c:613:11: error: no member named 'data' in 'struct grpc_event'
  if (ev->data.op_complete != GRPC_OP_OK) {
      ~~  ^
rb_call.c:613:31: error: use of undeclared identifier 'GRPC_OP_OK'
  if (ev->data.op_complete != GRPC_OP_OK) {
                              ^
rb_call.c:616:18: error: no member named 'data' in 'struct grpc_event'
             ev->data.op_complete);
             ~~  ^
3 errors generated.
make: *** [rb_call.o] Error 1

make failed, exit code 2
```

--- a/src/ruby/lib/grpc/version.rb
+++ b/src/ruby/lib/grpc/version.rb
@@ -29,5 +29,5 @@
 
 # GRPC contains the General RPC module.
 module GRPC
-  VERSION = '0.9.0'
+  VERSION = '0.9.2'
 end

